org-formation/720-client-vpn/vpn.njk

```diff
@@ -133,7 +145,7 @@ Resources:
     Properties:
       Description: {{ spoke }}-Subnet{{ subnet_id | last }}
       ClientVpnEndpointId: !Ref Endpoint
-      DestinationCidrBlock: {{ spoke_data.CIDR }}
+      DestinationCidrBlock: 0.0.0.0/0
```

Comment:
hmm, so we are changing this from being able to access specific spokes to being able to access anything?  seems too permissive to me.